src/txn/txn_rollback_to_stable.c

```diff
@@ -265,6 +239,323 @@ __rollback_check_if_txnid_non_committed(WT_SESSION_IMPL *session, uint64_t txnid
     return (found);
 }
 
+/*
+ * __rollback_col_ondisk_fixup_key --
+ *     Allocate tombstone and calls function add update to head of insert list.
+ */
+static int
+__rollback_col_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_COL *cip,
+  wt_timestamp_t rollback_timestamp, bool replace, uint64_t recno)
+{
+    WT_CELL *kcell;
+    WT_CELL_UNPACK_KV *unpack, _unpack;
+    WT_CURSOR *hs_cursor;
+    WT_CURSOR_BTREE *cbt;
+    WT_DECL_ITEM(hs_key);
+    WT_DECL_ITEM(hs_value);
+    WT_DECL_ITEM(key);
+    WT_DECL_RET;
+    WT_ITEM full_value;
+    WT_PAGE *page;
+    WT_UPDATE *hs_upd, *tombstone, *upd;
+    wt_timestamp_t hs_durable_ts, hs_start_ts, hs_stop_durable_ts, newer_hs_durable_ts;
+    uint64_t hs_counter, type_full;
+    uint32_t hs_btree_id;
+    uint8_t *p;
```

Comment:
What is p?

Reply:
p is a pointer to memory managed by the key item. Should I rename it to memp or key_memp or something similar?

Reply:
I do prefer memp.

Reply:
Fixed.